ios/RNSQIPCardEntry.m

```diff
@@ -162,6 +162,111 @@ - (dispatch_queue_t)methodQueue
+                 contactMap
+                 : (NSDictionary *)contactMap
+                 startCardEntryFlowWithResolver
+                 : (RCTPromiseResolveBlock)resolve
+                     rejecter
+                 : (RCTPromiseRejectBlock)reject)
+{
+    dispatch_async([self methodQueue], ^{
+        SQIPMoney *money = [[SQIPMoney alloc] initWithAmount:[moneyMap[@"amount"] longValue]
+                            currency:[RNSQIPBuyerVerification currencyForCurrencyCode:moneyMap[@"currencyCode"]]];
+
+        SQIPBuyerAction *buyerAction = nil;
+        if ([@"Store" isEqualToString:buyerActionString]) {
+            buyerAction = [SQIPBuyerAction storeAction];
+        } else {
+            buyerAction = [SQIPBuyerAction chargeActionWithMoney:money];
+        }
+
+        SQIPContact *contact = [[SQIPContact alloc] init];
+        contact.givenName = contactMap[@"givenName"];
+
+        if (![contactMap[@"familyName"] isEqual:[NSNull null]]) {
+            contact.familyName = contactMap[@"familyName"];
+        }
+
+        if (![contactMap[@"email"] isEqual:[NSNull null]]) {
+            contact.email = contactMap[@"email"];
+        }
+
+        if (![contactMap[@"addressLines"] isEqual:[NSNull null]]) {
+            contact.addressLines = contactMap[@"addressLines"];
+            NSLog(@"%@", contactMap[@"addressLines"]);
+        }
+
+        if (![contactMap[@"city"] isEqual:[NSNull null]]) {
+            contact.city = contactMap[@"city"];
+        }
+
+        if (![contactMap[@"region"] isEqual:[NSNull null]]) {
+            contact.region = contactMap[@"region"];
+        }
+
+        if (![contactMap[@"postalCode"] isEqual:[NSNull null]]) {
+            contact.postalCode = contactMap[@"postalCode"];
+        }
+
+        contact.country = [RNSQIPBuyerVerification countryForCountryCode:contactMap[@"countryCode"]];
+
+        if (![contactMap[@"phone"] isEqual:[NSNull null]]) {
+            contact.phone = contactMap[@"phone"];
+        }
+
+        self.locationId = locationId;
+        self.buyerAction = buyerAction;
+        self.contact = contact;
+
+        UIViewController *rootViewController = UIApplication.sharedApplication.keyWindow.rootViewController;
+
+    
+        SQIPVerificationParameters *params = [[SQIPVerificationParameters alloc] initWithPaymentSourceID:paymentSourceId
+                                                buyerAction:self.buyerAction
+                                                locationID:self.locationId
+                                                contact:self.contact];
+
+        if ([rootViewController isKindOfClass:[UINavigationController class]]) {
+            [rootViewController.navigationController popViewControllerAnimated:YES];
+        } else {
+            [rootViewController dismissViewControllerAnimated:YES completion:nil];
+        }
+
+        [SQIPBuyerVerificationSDK.shared verifyWithParameters:params
+            theme:self.theme
```

Comment:
`self.theme` need to be initialized, otherwise you will have exception.

Reply:
Is it not fixed yet?

Reply:
Yes. This one is fixed.